components/onboarding/OnboardingVisualStepper.tsx

```diff
@@ -0,0 +1,64 @@
+import StatusComplete from '../shared/icons/StatusComplete';
+import StatusCurrent from '../shared/icons/StatusCurrent';
+import StatusIncomplete from '../shared/icons/StatusIncomplete';
+
+const OnboardingVisualStepper = ({ step }: { step: number }) => {
+  const renderSteps = () => {
+    const RepeatedPatternIncomplete = (number: number) =>
+      Array(number).fill(
+        <>
+          <div className="border border-black-600 w-28"></div>
+          <StatusIncomplete />
+        </>
+      );
+
+    const RepeatedPatternComplete = (number: number) =>
+      Array(number).fill(
+        <>
+          <StatusComplete />
+          <div className="border border-primary-500 w-28"></div>
+        </>
```

Comment:
![image](https://github.com/DDVVPP/GitNote/assets/4108484/b2cf1102-9815-41b8-a983-63f2c6b24369)

If you need this to be a fragment, you can still set a key using 'React.Fragment'

```suggestion
     Array(number).fill(
        <React.Fragment key={number}>
          <div className="border border-black-600 w-28"></div>
          <StatusIncomplete />
        </React.Fragment>
      );

    const RepeatedPatternComplete = (number: number) =>
      Array(number).fill(
        <React.Fragment key={number}>
          <StatusComplete />
          <div className="border border-primary-500 w-28"></div>
        </React.Fragment>
```